src/interface.jl

```diff
@@ -111,3 +111,51 @@ function value!!(obj, F, x)
     copy!(obj.x_f, x)
     obj.f_calls .+= 1
 end
+
+function _clear_f!(d::NLSolversBase.AbstractObjective)
+    d.f_calls .= 0
+    if typeof(d.F) <: AbstractArray
+        d.F .= eltype(d.F)(NaN)
+    else
+        d.F = typeof(d.F)(NaN)
+    end
+    d.x_f .= eltype(d.x_f)(NaN)
+end
+
+function _clear_df!(d::NLSolversBase.AbstractObjective)
+    d.df_calls .= 0
+    d.DF .= eltype(d.DF)(NaN)
+    d.x_df .= eltype(d.x_df)(NaN)
+end
+
+function _clear_h!(d::NLSolversBase.AbstractObjective)
+    d.h_calls .= 0
+    d.H .= eltype(d.H)(NaN)
+    d.x_h .= eltype(d.x_h)(NaN)
+end
+
+function _clear_hv!(d::NLSolversBase.AbstractObjective)
+    d.hv_calls .= 0
+    d.Hv .= eltype(d.Hv)(NaN)
+    d.x_hv .= eltype(d.x_hv)(NaN)
+    d.v_hv .= eltype(d.v_h)(NaN)
+end
+
+clear!(d::NonDifferentiable)  = _clear_f!(d)
+
+function clear!(d::OnceDifferentiable)
+    _clear_f!(d)
+    _clear_df!(d)
+end
+
+function clear!(d::TwiceDifferentiable)
+    _clear_f!(d)
+    _clear_df!(d)
+    _clear_h!(d)
+end
+
+function clear!(d::TwiceDifferentiableHV)
```

Comment:
Any reason why this isn't tested as the only one?

Reply:
There are no existing tests for TwiceDifferentiableHV, so I didn't know how to create such an objective. I created an issue in #34.

Reply:
Ah, alright! I'll add a test asap.